compiler/src/vm/builtin_functions.rs

```diff
@@ -292,6 +298,59 @@ impl Heap {
         })
     }
 
+    fn list_create(&mut self, args: &[Pointer]) -> BuiltinResult {
+        unpack_and_later_drop!(self, args, |length: Int| {
+            let length = length.value.to_usize().unwrap();
+            let nothing = self.create_nothing();
+            Return(self.create_list(vec![nothing; length]))
+        })
+    }
+    fn list_get(&mut self, args: &[Pointer]) -> BuiltinResult {
+        unpack_and_later_drop!(self, args, |list: List, index: Int| {
+            let index = index.value.to_usize().unwrap();
```

Comment:
If `builtinListGet` is called with a negative number, we probably don't want to crash the entire VM/LSP with `unwrap`. Instead, we probably want to do something like this:

```suggestion
            let index = index.value.to_usize().map_err(|_| "builtinListGet needs a non-negative index")?;
```

The same applies to all the argument handling in this file.